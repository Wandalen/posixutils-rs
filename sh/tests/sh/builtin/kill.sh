kill -l

trap 'echo received SIGTERM' TERM
kill $$
kill -- $$
kill -TERM $$
kill -TERM -- $$
kill -s TERM $$
kill -s TERM -- $$
kill -0 $$

