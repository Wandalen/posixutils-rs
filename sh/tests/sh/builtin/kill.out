HUP
INT
QUIT
ILL
TRAP
ABRT
BUS
FPE
KILL
USR1
SEGV
USR2
PIPE
ALRM
TERM
CHLD
CONT
STOP
TSTP
TTIN
TTOU
URG
XCPU
XFSZ
VTALRM
PROF
SYS
received SIGTERM
received SIGTERM
received SIGTERM
received SIGTERM
received SIGTERM
received SIGTERM
